app/views/setup/discovery.html.slim

```diff
@@ -36,7 +36,7 @@ h1 Select nodes and roles
       i.fa.fa-times.fa-fw
       | Deselect all nodes
   .panel-body
-    p After choosing the master and clicking "Next" all the other selected nodes will be set to the worker role.
+    p After choosing the nodes and clicking "Next", all the selected nodes will be set to bootstrap.
```

Comment:
I think this  still does not reflect what we want the user to do. Maybe something like this.

"Select 1 node as "Master" node and one or more of the remaining nodes as "Worker" nodes to bootstrap the cluster"

Reply:
Maybe `Select at least one node as master and one or more of the remaining nodes as worker to  set up the cluster to bootstrap`? 

My English is not good, so... Let's wait for other opinions.

Reply:
Suggestion:

«
After selecting at least one node to be master, click "Select remaining nodes" and then "Next" to make all the others into workers.
»

Reply:
> After selecting at least one node to be master, click "Select remaining nodes" and then "Next" to make all the others into workers.

Doesn't it sound confusing having `to make all the others into workers` after `"Next"`?

Trying a quick fix...

`After selecting at least one node to be master, click "Select remaining nodes" to make all the others into workers and then "Next" to set up nodes to bootstrap.`

Reply:
Consider:

```
    p
      | Please choose an odd number of
      i<> Master
      | nodes, then click
      i<> "Select remaining nodes"
      | to assign the remaining nodes to the
      i<> Worker
      | role.
```

Reply:
I like @bear454 proposal. Let's go with it

Reply:
I like that. I was trying to minimize alteration, but this is better.